packages/bits/spec/components/repeat/repeat.e2e.ts

```diff
@@ -45,68 +39,63 @@ describe("USERCONTROL Repeat", () => {
         expect(await singleSelectList.isCompact()).toEqual(true);
     });
 
-    it("should not be 'striped'", async () => {
-        expect(await multiSelectList.isStriped()).toEqual(false);
-    });
-
-    it("should have the correct header", async () => {
-        expect(await multiSelectList.getHeaderText()).toBe("repeat header from template");
-    });
-
-    it("should apply the correct template", async () => {
-        const blueColorItem = multiSelectList.getItem(0);
-        expect(await blueColorItem.getText()).toEqual("blue");
-    });
-
     it("should allow multi selection of items", async () => {
         await multiSelectList.selectCheckbox(2);
         await multiSelectList.selectCheckbox(5);
-        expect(await element(by.id("nui-demo-multiselect-values")).getText()).toBe("[]");
+        expect(await colorSelection.getText()).toBe("[]");
         await multiSelectList.selectCheckbox(0);
-        expect(await element(by.id("nui-demo-multiselect-values")).getText()).toBe("[ { \"color\": \"blue\" } ]");
-    });
+        expect(await colorSelection.getText()).toBe("[ { \"color\": \"blue\" } ]");
 
-    it("should not allow selection change for disabled items", async () => {
-        const multiSelectListDisabledItems = Atom.find(RepeatAtom, "nui-demo-multi-repeat-disabled-items");
-        expect(await multiSelectListDisabledItems.getCheckbox(0).isDisabled()).toBe(true);
-        expect(await multiSelectListDisabledItems.getCheckbox(4).isDisabled()).toBe(true);
+        // Return to initial state
+        await multiSelectList.selectCheckbox(2);
+        await multiSelectList.selectCheckbox(5);
+        await multiSelectList.selectCheckbox(0);
+        expect(await colorSelection.getText()).toBe("[ { \"color\": \"yellow\" }, { \"color\": \"black\" } ]");
+        await colorSelection.click();
     });
 
     describe("keyboard navigation", () => {
-        beforeEach(async () => {
-            await browser.refresh();
-            await Helpers.pressKey(Key.TAB);
-        });
-
         it("should allow check/uncheck items using ENTER", async () => {
+            await Helpers.pressKey(Key.TAB);
             await Helpers.pressKey(Key.ENTER);
             multiSelectList.getCheckbox(0);
-            expect(await element(by.id("nui-demo-multiselect-values")).getText()).toBe("[ { \"color\":" +
+            expect(await colorSelection.getText()).toBe("[ { \"color\":" +
                 " \"yellow\" }, { \"color\": \"black\" }, { \"color\": \"blue\" } ]");
 
             await Helpers.pressKey(Key.ENTER);
-            expect(await element(by.id("nui-demo-multiselect-values")).getText()).toBe("[ { \"color\":" +
+            expect(await colorSelection.getText()).toBe("[ { \"color\":" +
                 " \"yellow\" }, { \"color\": \"black\" } ]");
+
+            // Return to initial state
+            await colorSelection.click();
```

Comment:
Does this do something? I think `colorSelection` is just a `span`.

Reply:
Right, Blake, that is the same as before - it clicks on header text to not highlight a checkbox.